sigle/src/modules/settings/SettingsForm.tsx

```diff
@@ -128,15 +128,30 @@ export const SettingsForm = ({ settings, username }: SettingsFormProps) => {
       const website = formik.values.siteUrl;
 
       formik.values.siteTwitterHandle = handle.split('/').pop() as string;
-      newSettings.siteTwitterHandle = handle.split('/').pop() as string;
 
-      if (handle.includes('@')) {
-        formik.values.siteTwitterHandle = handle.replace('@', '');
-        newSettings.siteTwitterHandle = handle.replace('@', '');
-      }
+      formik.values.siteTwitterHandle = handle.replace('@', '');
+      newSettings.siteTwitterHandle = handle.replace('@', '');
```

Comment:
this line is needed to prevent an extra `@` appearing on blog page
![Screenshot 2022-05-03 at 11 05 11](https://user-images.githubusercontent.com/65421744/166450636-19c4430f-be88-482a-8cc6-825aa0413f61.png)